parsers/src/main/java/org/iqu/parsers/Parser.java

```diff
@@ -9,16 +10,16 @@
  */
 public interface Parser<T> {
 
-  /**
-   * Parses the RSS at the given URL and returns the content as a List of
-   * objects of type T.
-   * 
-   * @param sourceURL
-   *          the link to the RSS feed
-   * @param encoding
-   *          the encoding of the source material
-   * @return parsed material as a List of objects
-   */
-  public List<T> readFeed(String sourceURL, String encoding);
+	/**
+	 * Parses the RSS at the given URL and returns the content as a List of
+	 * objects of type T.
+	 * 
+	 * @param sourceURL
+	 *            the link to the RSS feed
+	 * @param encoding
+	 *            the encoding of the source material
+	 * @return parsed material as a List of objects
+	 */
+	public List<T> readFeed(String sourceURL, String encoding) throws ParseException;
```

Comment:
Is it still needed to throw this type of exception? In case it ease please create a custom exception e.g. org.iqu.exceptions.ParserException and wrap the exception throwned with it